app/internal/json_data_loader.py

```diff
@@ -5,8 +5,8 @@
 from loguru import logger
 from sqlalchemy.orm import Session
 
-from app.database.models import Base, Quote, Zodiac
-from app.internal import daily_quotes, zodiac
+from app.database.models import Base, Quote, Zodiac, InternationalDays
+from app.internal import daily_quotes, zodiac, international_days
```

Comment:
ABC. Use git precommit hooks to get rid of all the style comments

Reply:
fixed